tests/acceptance/users-edit-responsibilities-test.js

```diff
@@ -9,7 +9,16 @@ module("Acceptance | users edit responsibilities", function (hooks) {
   setupMirage(hooks);
 
   hooks.beforeEach(async function () {
-    this.user = this.server.create("user");
+    this.user = this.server.create("user", {
+      isSuperuser: true,
+      firstName: "Admin FirstName",
+      lastName: "Admin LastName",
+    });
+    this.secondUser = this.server.create("user", {
+      supervisorIds: [this.user.id],
+      firstName: "FirstName",
+      lastName: "LastName",
+    });
```

Comment:
Nitpick: You could rename `this.user` --> `this.admin` and `this.secondUser` --> `this.user`. That way it would be more obvious what's the difference between those two users you've created. In this case the test is quite small and easy to grasp, but if it grows it becomes messy quite fast.